modules/utxo-lib/src/bitgo/transaction.ts

```diff
@@ -65,7 +77,7 @@ export function setTransactionBuilderDefaults(
       txb.setVersion(version);
       break;
     case networks.zcash:
-      (txb as ZcashTransactionBuilder).setDefaultsForVersion(network, version);
+      (txb as unknown as ZcashTransactionBuilder).setDefaultsForVersion(network, version);
```

Comment:
External users complain about the use of `unknown` and `any`. Is it possible to identify this?

Reply:
I think I can remove this by refactoring